### PR TITLE
NsTimePicker drop direction prop

### PR DESCRIPTION
**Pull request by tommaso-ascani**
Added new prop for NsTimePicker to allow choosing the dropdown menù direction directly to the component and fixed some syntax errors on prop validators.

### CODE REVIEW

File: src/lib-components/NsTimePicker.vue [hunk 1]
```diff
@@ -55,6 +55,10 @@ export default {
       type: String,
       default: "",
     },
+    dropDirection: {
+      type: String,
+      default: "down"
+    }
```
Comment (andre8244) [inline src/lib-components/NsTimePicker.vue:61]: Let's add a validator to ensure that `up| down | auto` is passed to the component. Example: https://github.com/NethServer/ns8-ui-lib/blob/main/src/lib-components/NsComboBox.vue#L254

Comment (andre8244) [reply src/lib-components/NsTimePicker.vue:62]: I noticed an error in the example I linked above :scream: 

This is correct istead: https://github.com/NethServer/ns8-ui-lib/blob/main/src/lib-components/NsComboBox.vue#L249

After fixing `dropDirection` validator, could you also fix NsComboBox `tooltipDirection` validator in a separate commit please?

Comment (tommaso-ascani) [reply src/lib-components/NsTimePicker.vue:62]: Separate commit in another pr or here?

Comment (andre8244) [reply src/lib-components/NsTimePicker.vue:62]: Here is fine

**Pull request merged**